src/script/components/GuestModeToggle.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import {registerReactComponent} from 'Util/ComponentUtil';
+import cx from 'classnames';
+import {t} from 'Util/LocalizerUtil';
```

Comment:
```suggestion
import React from 'react';
import cx from 'classnames';

import {registerReactComponent} from 'Util/ComponentUtil';
import {t} from 'Util/LocalizerUtil';
```

Reply:
I don't see any value in this import separation stuff TBH.

Reply:
It's just one click

Reply:
I rarely see myself asking myself what is an external import or local import. Anyways, merged.